### PR TITLE
{bio}[foss/2020a] cisDIVERSITY v1.1

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)
resolves https://github.com/vscentrum/vsc-software-stack/issues/435

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

**Comment by jfgrimm** [comment]
@pavelToman this toolchain is [considered unsupported](https://docs.easybuild.io/policies/toolchains/?h=toolchain+support#current-situation)

please either bump to at least 2021b, or close the PR

File: easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a.eb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'cisDIVERSITY'
+version = '1.1'
```
Comment (boegel) [inline easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a.eb:4]: @pavelToman this one should be removed since it lacks the `versionsuffix`?

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a-Python-2.7.18.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+name = 'cisDIVERSITY'
+version = '1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/NarlikarLab/cisDIVERSITY/'
+description = """A module discovery tool used for finding diverse sequence architectures, 
+each one characterized by presence or absence of de novo motifs."""
+
+toolchain = {'name': 'foss', 'version': '2020a'}
```
Comment (boegel) [inline easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a-Python-2.7.18.eb:11]: @pavelToman Can you clarify why you're using this ancient toolchain here?

Very soon that will no longer be installable (like on RHEL9)

Comment (pavelToman) [reply easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a-Python-2.7.18.eb:11]: It needs python 2: https://github.com/vscentrum/vsc-software-stack/issues/435#issuecomment-2405207422

Comment (pavelToman) [reply easybuild/easyconfigs/c/cisDIVERSITY/cisDIVERSITY-1.1-foss-2020a-Python-2.7.18.eb:11]: Maybe the foss/2021b could be used, there is `SciPy-bundle-2021.10-foss-2021b-Python-2.7.18` with numpy for python 2. I am gonna try it

**Comment by pavelToman** [comment]
New PR: 
- https://github.com/easybuilders/easybuild-easyconfigs/pull/21692

**Pull request closed without merging**